portal/emailMessages_new.py

```diff
@@ -69,120 +69,6 @@ def emailChangeNotificationEmail(request, new_email):
         'subject': emailSubjectPrefix() + " : Email address changed",
         'message': ("Someone has tried to change the email address of your account. If this was " +
                     "not you, please get in contact with us via " +
-                    request.build_absolute_uri(reverse('contact')) + "." +
+                    request.build_absolute_uri(reverse('help_new')) + "#contact ." +
                     emailBodySignOff(request)),
     }
-
-
-def joinRequestPendingEmail(request, pendingAddress):
-    return {
```

Comment:
These were unused email functions in the new file, thus decreasing coverage :-1: 
So I got rid of them.